azure-pipelines.yml

```diff
@@ -65,79 +79,110 @@ jobs:
 
 - job: dockerBuildAndPush
   displayName: Build and push docker containers to the acr
+  variables:
+    semVer: $[ dependencies.version.outputs['setVersionStep.semVer'] ]
   condition: and(succeeded(), not(eq(variables['build.reason'], 'PullRequest')))
-  dependsOn: buildAndTest
+  dependsOn: 
+    - version
+    - buildAndTest
   steps:
-
   - task: DockerInstaller@0
     inputs:
       dockerVersion: $(dockerVersion)
 
   - task: DockerCompose@0
     displayName: 'Build: Dev'
+    condition: and(succeeded(), ne(variables['build.reason'], 'PullRequest'))
     inputs:
       containerregistrytype: 'Azure Container Registry'
       azureSubscription: 'NHSAPP-BuyingCatalogue (Non-Prod)'
       azureContainerRegistry: '{"loginServer":"gpitfuturesdevacr.azurecr.io", "id" : "/subscriptions/7b12a8a2-f06f-456f-b6f9-aa2d92e0b2ec/resourceGroups/gpitfutures-dev-rg-acr/providers/Microsoft.ContainerRegistry/registries/gpitfuturesdevacr"}'
-      dockerComposeFile: '**/docker-compose-build.yml'
+      dockerComposeFile: '**/docker-compose.yml'
       action: 'Build services'
+      additionalImageTags: '$(semVer)'
       includeSourceTags: true
 
   - task: DockerCompose@0
     displayName: 'Push: Dev'
+    condition: and(succeeded(), ne(variables['build.reason'], 'PullRequest'))
     inputs:
       containerregistrytype: 'Azure Container Registry'
       azureSubscription: 'NHSAPP-BuyingCatalogue (Non-Prod)'
       azureContainerRegistry: '{"loginServer":"gpitfuturesdevacr.azurecr.io", "id" : "/subscriptions/7b12a8a2-f06f-456f-b6f9-aa2d92e0b2ec/resourceGroups/gpitfutures-dev-rg-acr/providers/Microsoft.ContainerRegistry/registries/gpitfuturesdevacr"}'
-      dockerComposeFile: '**/docker-compose-build.yml'
+      dockerComposeFile: '**/docker-compose.yml'
       action: 'Push services'
+      additionalImageTags: '$(semVer)'
       includeSourceTags: true
 
+  - task: HelmInstaller@1
```

Comment:
think this needs condition: and(succeeded(), ne(variables['build.reason'], 'PullRequest')) too

Reply:
No need, as the job that this task falls under already has that condition - have a look [here](https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_build/results?buildId=6810&view=results)